tutor/src/components/tabs.jsx

```diff
@@ -56,9 +56,7 @@ const Tabs = ({
     ) {
       const ev = new FakeEvent;
       onSelect(activeIndex, ev);
-      if (ev.isDefaultPrevented()) {
-        selectTabIndex(activeIndex);
-      }
+      selectTabIndex(activeIndex);
     }
```

Comment:
we need to leave the defaultPrevented check.  That allows onSelect handlers to veto a tab change.

Reply:
oh yeah, forgot to put it back. I was doing some testing. thanks nathan.